frontend/src/lib/components/icons.scss

```
@@ -22,3 +21,19 @@
         }
     }
 }
+
+.icon-count-bubble {
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    line-height: 16px;
+    top: -0.3rem;
+    right: -0.3rem;
+    color: var(--white);
+    background: var(--primary);
+    font-size: 0.6rem;
```

These values are a bit off in px terms, as 0.6rem is typically 9.6px (unless user changes root font size from 16px). They should generally be multiples of 1/16.
```suggestion
    font-size: 0.5625rem;
```